tests/results/result_actions/IgnoreResultActionTest.py

```diff
@@ -101,6 +101,16 @@ def test_ignore(self):
             with open(f_a, 'r') as f:
                 self.assertEqual(file_diff_dict[f_a].modified, f.readlines())
 
+            # Apply a third patch, old patch has to stay!
+            uut.apply(Result.from_values('bear', 'msg', f_a, 3),
+                      file_dict, file_diff_dict, 'jinja2')
+            self.assertEqual(
+                file_diff_dict[f_a].modified,
+                ['1  /* Ignore else */\n', '2  // Ignore origin\n',
```

Comment:
these comments are obviously invalid for jinja, so it does not make sense to add your test to this method.  Create a new test method for jinja.

Reply:
You can use simply use [get_ignore_comment](https://github.com/coala/coala/blob/3c3fab54063f05960a3862ea3b2c83174341a2cf/coalib/results/result_actions/IgnoreResultAction.py#L81) to directly check whether your definition is working or not and accordingly write a test for that.